charts/posthog/tests/events-deployment.yaml

```diff
@@ -32,3 +32,63 @@ tests:
           count: 1
       - isKind:
           of: Deployment
+
+  - it: spec.template.spec.containers[0].env should have the default statsd env variables when `prometheus-statsd-exporter.enabled` is `true`
```

Comment:
If the tests for the deployments are the same, would it be better to just define them once and test across all deployments? Or is this too rigid?

Reply:
Ok, let me change it